### PR TITLE
Spotlight update for August 2024 - images, please merge to amada-spotlight-aug-2024

### PR DESCRIPTION
**Pull request by amada-ech**
## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/community/spotlight/meagan-palmer.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+id: meagan-palmer
+title: Meagan Palmer
+description: |
+  I first started using dbt in 2016 or 2017 (I can't remember exactly). Since then, I have moved into data and analytics consulting and have dipped in and out of the dbt Community. 
+  Late last year, I started leading dbt Cloud training courses and spending more time in the <a href="https://www.getdbt.com/community/join-the-community/“ rel="noopener noreferrer" target="_blank">dbt Slack</a>.
```
Comment (mirnawong1) [inline website/docs/community/spotlight/meagan-palmer.md]: i think this is what could be causing the error - should it be:  `<a target="_blank" rel="noopener noreferrer" href="https://www.getdbt.com/community/join-the-community/">dbt Slack</a>` instead?

Comment (mirnawong1) [inline website/docs/community/spotlight/meagan-palmer.md]: i think this is what could be causing the error - should it be this instead?
```suggestion
  Late last year, I started leading dbt Cloud training courses and spending more time in the <a target="_blank" rel="noopener noreferrer" href="https://www.getdbt.com/community/join-the-community/">dbt Slack</a>. 
```

File: website/docs/community/spotlight/meagan-palmer.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+id: meagan-palmer
+title: Meagan Palmer
+description: |
+  I first started using dbt in 2016 or 2017 (I can't remember exactly). Since then, I have moved into data and analytics consulting and have dipped in and out of the dbt Community. 
+  Late last year, I started leading dbt Cloud training courses and spending more time in the <a href="https://www.getdbt.com/community/join-the-community/“ rel="noopener noreferrer" target="_blank">dbt Slack</a>. 
+  In consulting, I get to use a range of stacks. I've used dbt with Redshift, Snowflake, and Databricks in production settings with a range of loaders & reporting tools, and I've been enjoying using DuckDB for some home experimentation. 
+  To share some of the experiences, I regularly post to LinkedIn and have recently started <a href="https://www.linkedin.com/newsletters/analytics-engineering-today-7210968984693690370/“ rel="noopener noreferrer" target="_blank">Analytics Engineering Today</a>, a twice monthly newsletter about dbt in practice.
```
Comment (mirnawong1) [inline website/docs/community/spotlight/meagan-palmer.md]:
```suggestion
  To share some of the experiences, I regularly post to LinkedIn and have recently started <a target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/newsletters/analytics-engineering-today-7210968984693690370/“>Analytics Engineering Today</a>, a twice monthly newsletter about dbt in practice.
```

File: website/docs/community/spotlight/meagan-palmer.md (hunk 1 above)
Comment (john-rock) [reply website/docs/community/spotlight/meagan-palmer.md]: I think you're correct @mirnawong1. It looks like that fancy quote after the href is causing a parsing error in the build.
Spotlight_update_for_August_2024_-_images__please_merge_to_amada-spotlight-aug-2024

**Review by john-rock — COMMENTED** [review]
Hey @amada-ech I believe we just need to replace the `“` with `"` at the end of the href values to get this deploy to build successfully.

Everything else looks good but I'll give it another look once those adjustments are made.

**Review by john-rock — APPROVED** [review]
lgtm after the latest edits!

**Comment by john-rock** [comment]
Merged for @amada-ech since she did not have the permissions to do so.

**Pull request merged**